lib/base.js

```diff
@@ -52,13 +52,49 @@ class Base {
       })
   }
 
+  _getPages (body) {
```

Comment:
add a jsdoc here

just type 
```
/**
```
into webstorm, and it'll make most of it for you